thunder/recipes/hf_transformers.py

```diff
@@ -41,13 +41,14 @@ class HFTransformers(BaseRecipe):
     def __init__(
         self,
         show_progress=False,
-        fuser="nvfuser",
         interpreter="thunder.jit",
         plugins=None,
     ):
-        super().__init__(show_progress=show_progress, fuser=fuser, interpreter=interpreter, plugins=plugins)
+        super().__init__(show_progress=show_progress, interpreter=interpreter, plugins=plugins)
+
         # for kv-cache inplace ops
         self.inplace_index_copy_transform = InplaceIndexCopyTransform()
+        self.executors.append(InplaceIndexCopyTransform().executor.name)
```

Comment:
would it work if we used the executor here instead of the name? It seems the global lookup is causing trouble somewhere?